templates/_linux.yaml

```diff
@@ -32,6 +33,11 @@ objects:
     template:
       spec:
         domain:
+{%- if item.iothreads == true %} {{''}}
```

Comment:
Why not just?

```suggestion
{%- if item.iothreads %}
```

Reply:
it's just double check - "never trust data input". In a case if we choose something strange in playbook.

Reply:
That is our responsibility, the templates here and the parameters are code we own and maintain. This check specifically will give you no warning anyway.